src/common/AndroidUtils.ts

```diff
@@ -550,12 +550,20 @@ export class AndroidSDKUtils {
     public static launchNativeApp(
         compName: string,
         projectDir: string,
+        appBundlePath: string | undefined,
         targetApp: string,
         targetAppArguments: LaunchArgument[],
         launchActivity: string,
         emulatorPort: number
     ): Promise<boolean> {
         try {
+            if (appBundlePath && appBundlePath.trim().length > 0) {
+                const installCommand = `${
+                    AndroidSDKUtils.ADB_SHELL_COMMAND
+                } -s emulator-${emulatorPort} install -r -t '${appBundlePath.trim()}'`;
```

Comment:
should we have the `-t` here, always?

Reply:
I *think* probably so. If you look at the [`android:testOnly` documentation](https://developer.android.com/guide/topics/manifest/application-element), at the bottom it states:

> Android Studio automatically adds this attribute when you click Run .

Which leads me to believe that the kinds of apps that we will end up using here will likely fall into the `testOnly` bucket. It's not a certainty, but I think going with `-t` always at this point is a reasonable bet.